queries.sql

```diff
@@ -38,4 +38,4 @@ SELECT AVG(weight_kg) FROM animals;
 
 SELECT neutered, SUM(escape_attempts) FROM animals GROUP BY neutered;
 
-SELECT name, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY name;
+SELECT name, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY name;
```

Comment:
* According to the [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md#:~:text=Inside%20a%20transaction,queries.sql%20file.) you are to 
* `verify that change was made`
* roll back the change
* `verify that change was undone`
* Commit the transaction.
* Verify that change was made and persists after commit. 
*Inside a transaction:

> Delete all animals born after Jan 1st, 2022.
> Create a savepoint for the transaction.
> Update all animals' weight to be their weight multiplied by -1.
> Rollback to the savepoint
> Update all animals' weights that are negative to be their weight multiplied by -1.
> Commit transaction
* Below is a screenshot of the highlighted requirements that are missing kindly write queries to answer this conditions 
![image](https://user-images.githubusercontent.com/52497246/228217906-a0b63f95-c0b1-4c87-9fc9-339dff87aa6f.png)

Reply:
So I have to insert transaction commands in the queries.sql? Btw I have also verify that changes in the screenshots added in the PR description.

Reply:
And I have written all the queries in the project requirements except transaction commands. Can you tell me specifically what I missing?